README.md

```diff
@@ -38,34 +38,27 @@ Zim has many modules! Enable as many or as few as you'd like.
 Installation
 ------------
 Installing Zim is easy. If you have a different shell framework installed (like oh-my-zsh or prezto),
-*uninstall those first to prevent conflicts*. It can be installed manually by following the instructions below:
+**uninstall those first to prevent conflicts**. Then put following instructions at the end of your `~/.zshrc`:
 
-1. In a Zsh shell, clone the repository:
-  ```
-  git clone --recursive https://github.com/zimfw/zimfw.git ${ZDOTDIR:-${HOME}}/.zim
-  ```
+  ```bash
+  # Zim initializition
+  export ZIM_HOME="$HOME/.zim"
 
-2. Paste this into your terminal to prepend the initialization templates to your configs:
-  ```
-  setopt EXTENDED_GLOB
-  for template_file in ${ZDOTDIR:-${HOME}}/.zim/templates/*; do
-    user_file="${ZDOTDIR:-${HOME}}/.${template_file:t}"
-    touch ${user_file}
-    ( print -rn "$(<${template_file})$(<${user_file})" >! ${user_file} ) 2>/dev/null
-  done
-  ```
+  if [ ! -f "$ZIM_HOME/init.zsh" ]; then
+    echo "Installing zim"
+    git clone --recursive https://github.com/zimfw/zimfw.git $ZIM_HOME
+    cat $ZIM_HOME/templates/zlogin >> $HOME/.zlogin
+    source $HOME/.zlogin
+  fi
 
-3. Set Zsh as the default shell:
-  ```
-  chsh -s =zsh
-  ```
+  zmodules=(git git-info prompt completion syntax-highlighting autosuggestions)
+  zprompt_theme='steeef'
+  zhighlighters=(main brackets cursor)
```

Comment:
These 3 config lines are meant to be in the .zimrc file...

Reply:
What if there was no .zimrc? Settings could be added to .zshrc, or if not present, default functionality should execute anyway. Just thinking out load!

Reply:
I would prefer to keep zim settings separate, if for no other reason, so it's cleaner to switch to another plugin system.

Reply:
Modification on `.zshrc` is always need anyway. Deleting one line of code is no difference from deleting a block of continuous code. 

I actually prefer not to touch `.zlogin` at all and in my own setup I write my own 'zmanage reload' command to reload the `cache` when I needed it since tmux always open a login shell on split, and it slows down prompt about `50ms` on old server machines.